src/store/index.js

```diff
@@ -30,6 +32,14 @@ export default new Vuex.Store({
 			validateType(status, 'boolean', 'backdrop.status');
 			state.backdrop = status;
 		},
+		SET_INFO(state, status) {
+			validateType(status, 'boolean', 'message.status');
+			state.message = status;
+		},
+		SET_INFO_MESSAGE(state, text) {
+			validateType(text, 'string', 'message_text.status');
```

Comment:
third parameter, is just for human readability - I would change it to `info_message`. You don't have status on string :)